src/app/content/components/BuyBook.tsx

```diff
@@ -33,6 +37,7 @@ const BuyBookText = styled.span`
   margin: 0;
   font-weight: 600;
   line-height: 1.9rem;
+  overflow: hidden;
```

Comment:
This seems to be unnecessary

Reply:
It isn’t, prevents the span from being tabable on firefox.

Reply:
Is this span need for something then?

Reply:
i tried removing it and it looked the same, so i'm pushing that, karina can correct me later if it was needed for something